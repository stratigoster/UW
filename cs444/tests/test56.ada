-- no errors here
PACKAGE main IS
  a,b,c,d,e,f,g : integer := 10;
begin
  declare
    c : boolean := false;
    function foo(a:integer; b:float; c:boolean) return integer is
    begin
      c := a > a or else b=b;
    end foo;
    i : integer := foo(c=>true,a=>2,b=>3.0) + foo(b=>4.5,c=>false,10); -- thus stuff works!
  begin
    for x in 1..5 loop 
      i := i + 1;
    end loop;
    null;
  end;
END main;
