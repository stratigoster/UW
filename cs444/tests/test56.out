***ERROR***:11:46:	No compatible function found
***ERROR***:11:71:	No compatible function found

------
Compilation error
------
